posts/2022-03-17-graphql-22004-beta.adoc

```diff
@@ -0,0 +1,327 @@
+
+To enable the new beta features in your app, add them to your `server.xml`:
+
+[source, xml]
+----
+<featureManager>
+  <feature>mpGraphQL-2.0</feature>
+</featureManager>
+----
+
+[#java18]
+== Java 18 Support
+
+Java 18 is coming soon, and with it the following features and changes:
+
+* link:https://openjdk.java.net/jeps/400[UTF-8 by Default]
+* link:https://openjdk.java.net/jeps/408[Simple Web Server]
+* link:https://openjdk.java.net/jeps/413[Code Snippets in Java API Documentation]
+* link:https://openjdk.java.net/jeps/416[Reimplement Core Reflection with Method Handles]
+* link:https://openjdk.java.net/jeps/417[Vector API (Third Incubator)]
+* link:https://openjdk.java.net/jeps/418[Internet-Address Resolution SPI]
+* link:https://openjdk.java.net/jeps/419[Foreign Function & Memory API (Second Incubator)]
+* link:https://openjdk.java.net/jeps/420[Pattern Matching for switch (Second Preview)]
+* link:https://openjdk.java.net/jeps/421[Deprecate Finalization for Removal]
+
+By taking advantage of trying out the new changes in Java 18 now, you get more time to review your applications, microservices and runtime environments and be a step ahead when it becomes generally available.
+
+To give this a try today, link:https://jdk.java.net/18/[download] the early access release of Java 18, <<run, install 22.0.0.4-beta>> version of Open Liberty, edit your Liberty server's link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html#server-env[server.env file] to point `JAVA_HOME` to your Java 18 installation and start testing!
+
+For more information on Java 18, please visit the Java 18 link:https://jdk.java.net/18/release-notes[release notes page], link:https://download.java.net/java/early_access/jdk18/docs/api/[API Javadoc page] or link:https://jdk.java.net/18/[download page].
+For more information on Open Liberty, please visit our link:{url-prefix}/docs[documentation page].
+
+If you find anything amiss, <<feedback, let us know>>.
+
+[#security_caches]
+== Distributed Security Caches
+
+Distributed security cache support has been introduced so that multiple Liberty servers can share caches via a `JCache` provider.
+Prior to this feature, the authentication (subject) and logged out cookie caches were restricted to be local and in-memory. Multiple servers were unable to benefit from their peers' caches and each server started with a cold cache.
+
+As part of this feature both caches can now both be stored in a distributed `JCache`. This can improve performance and failure recovery, reduce the load on backend user registries and improve the security posture of the server.
```

Comment:
```suggestion
As part of this feature, both caches can be stored in a distributed `JCache` provider. This can improve performance and failure recovery, reduce the load on backend user registries, and improve the security posture of the server.
```